man/rgblink.5

```diff
@@ -86,6 +86,19 @@ If the bank has never been active thus far, the
 .Dq current address
 defaults to the beginning of the bank
 .Pq e.g. Ad $4000 No for Ic ROMX No sections .
+.Pp
+Instead of giving a bank number, the keyword
+.Ic FLOATING
+can be used instead; this binds the subsequent sections to no bank in particular (except if the type only contains a single bank, in which case
+.Ic FLOATING
+has no effect).
```

Comment:
```suggestion
Instead of giving a bank number, the keyword
.Ic FLOATING
can be used instead; this sets the type of the subsequent sections without binding them to a particular bank.
(If the type only allows a single bank, e.g.
.Ic ROM0 ,
then
.Ic FLOATING
is valid but redundant and has no effect.)
```